guides/v2.2/javascript-dev-guide/widgets/widget_quickSearch.md

```diff
@@ -37,6 +40,19 @@ Attaches the `autocomplete` attribute to the search field.
 
 **Accepted values**: `off`, `on`
 
+### `destinationSelector` {#q_destinationSelector}
+The element's selector where the results will be added.
+
+**Type**: String
```

Comment:
Please, provide the default value as well. Since it's not specified in the default options, it's supposed to be `null`.

Reply:
Also, make sure that we provide the default value for all entries. Thank you!